src/calculateRentalCost.js

```diff
@@ -26,9 +26,23 @@
  *
  * @return {number}
  */
+const LONG_TERM = 7;
+const LONG_TERM_DISCOUNT = 50;
+const SHORT_TERM = 3;
+const SHORT_TERM_DISCOUNT = 20;
 
 function calculateRentalCost(days) {
-  // write code here
+  const BASE_PRICE = days * 40;
```

Comment:
This "40" better to move out like SHORT_TERM and all other valuables you did before. :)
"BASE_PRICE", because it is within the function, the name should be in lowercase, but I'm not sure.

Reply:
that's right